ci-images/Dockerfile.base

```diff
@@ -1,15 +1,16 @@
-FROM quay.io/centos/centos:stream9
+FROM quay.io/centos/centos:stream8
```

Comment:
Is there a reason why we used `centos:stream9` initially? @osherdp Will moving back to `stream8` cause any issues?

Reply:
as long as CI passes it should be fine to downgrade
I think we had an effort moving to stream9/rhel9 that had to stop because of lack of support in D/S base images. We probably haven't reverted it completely ever-since we discovered this blocker

Reply:
Gotcha that's good to know, thanks Osher!